index.js

```diff
@@ -0,0 +1,95 @@
+const playerImg = document.querySelector(".player img")
+const playerName = document.querySelector(".player h2")
+const playerNick = document.querySelector(".player em")
+const playerLikes = document.querySelector(".likes")
+const playerGoalsUl = document.querySelector("#goals")
+const likeBtn = document.querySelector(".like-button")
+const newGoalForm = document.querySelector("#new-goal-form")
+
+const handleNewGoalFormSubmit = event => {
+	event.preventDefault()
+	const link = event.target.link.value
+	const description = event.target.description.value
+	const newGoal = {
+		playerId: playerName.dataset.id,
```

Comment:
The `playerId` key needs to be a number in order for `json-server` to correctly associate the goal with the player, so you'd need to parseInt here:

```js
playerId: parseInt(playerName.dataset.id),
```